cli/src/utils/initIAppWorkspace.js

```diff
@@ -94,42 +108,51 @@ async function copyChosenTemplateFiles({
   // rename _.gitignore (npm does not allow publishing files named .gitignore in a package)
   await fs.rename('_.gitignore', '.gitignore');
 
-  // transform template: remove unwanted feature code inside " // <<feature>> ... // <</feature>>" tags
-  const code = (await fs.readFile(srcFile)).toString('utf8');
-  let modifiedCode = code;
-  if (!useArgs) {
-    modifiedCode = modifiedCode.replaceAll(
-      / *\/\/ <<args>>\n((.*)\n)*? *\/\/ <<\/args>>\n/g,
-      ''
-    );
-  }
-  if (!useProtectedData) {
-    modifiedCode = modifiedCode.replaceAll(
-      / *\/\/ <<protectedData>>\n((.*)\n)*? *\/\/ <<\/protectedData>>\n/g,
-      ''
-    );
-  }
-  if (!useInputFile) {
-    modifiedCode = modifiedCode.replaceAll(
-      / *\/\/ <<inputFile>>\n((.*)\n)*? *\/\/ <<\/inputFile>>\n/g,
-      ''
-    );
-  }
-  if (!useRequesterSecret) {
-    modifiedCode = modifiedCode.replaceAll(
-      / *\/\/ <<requesterSecret>>\n((.*)\n)*? *\/\/ <<\/requesterSecret>>\n/g,
-      ''
-    );
-  }
-  if (!useAppSecret) {
-    modifiedCode = modifiedCode.replaceAll(
-      / *\/\/ <<appSecret>>\n((.*)\n)*? *\/\/ <<\/appSecret>>\n/g,
-      ''
-    );
+  for (const srcFile of srcFiles) {
+    // transform template: remove unwanted feature code inside " // <<feature>> ... // <</feature>>" tags
+    const code = (await fs.readFile(srcFile)).toString('utf8');
+    let modifiedCode = code;
+    if (!useArgs) {
+      modifiedCode = modifiedCode.replaceAll(
+        / *(\/\/|#) <<args>>\n((.*)\n)*? *(\/\/|#) <<\/args>>(\n)?/g,
```

Comment:
- added `#` with `//` comment tag for python
- final `\n` is optional to cover the case of closing tag ending the file without a new line